emcc.py

```diff
@@ -1116,9 +1126,9 @@ def check(input_file):
 
       if shared.Settings.BINARYEN:
         # set file locations, so that JS glue can find what it needs
-        shared.Settings.WASM_TEXT_FILE = os.path.basename(wasm_text_target)
-        shared.Settings.WASM_BINARY_FILE = os.path.basename(wasm_binary_target)
-        shared.Settings.ASMJS_CODE_FILE = os.path.basename(asm_target)
+        shared.Settings.WASM_TEXT_FILE = get_subresource_location(wasm_text_target)
```

Comment:
Just went ahead and tested my change. (I was expecting a complicated and/or long setup process based on past experience installing from source with the SDK, but in retrospect I think it was just compiling clang that would take a long time.)

It fails right here, apparently because the wasm, wast, and asm.js don't exist yet. How should I handle this?